index.html

```diff
@@ -36,98 +36,95 @@ <h2>Brigade is a tool for running scriptable, automated tasks in the cloud &mdas
         <div class="medium-7 columns">
 
           <h2>Simple, powerful pipes</h2>
-          <p>Each project gets a <code>brigade.js</code> config file, which is where you can write dynamic, interwoven pipelines and tasks for your Kubernetes cluster. Leave your YAML at home!</p>
+          <p>Each project uses a script file written in either JavaScript or TypeScript, which is where you can write dynamic, interwoven pipelines and tasks to handle inbound events. Leave your YAML at home!</p>
 
           <ul class="example-orbit-content" data-orbit>
             <li data-orbit-slide="headline-1">
               <pre>
-<em>// Run unit tests for a Github push</em>
-const { events, Job , Group} = require("brigadier");
-const dest = "$GOPATH/src/github.com/technosophos/ulid";
-
-events.on("push", (e, p) => {
-  console.log(e.payload)
-  var gh = JSON.parse(e.payload)
-  var test = new Job("test", "golang:1.9")
-  test.tasks = [
-    "mkdir -p " + dest,
-    "cp -a /src/* " + dest,
-    "cd " + dest,
-    "go get -u github.com/golang/dep/cmd/dep",
-    "dep ensure",
-    "make test"
-  ];
-  test.run()
-});
+<em>// Run unit tests for a Github push using TypeScript</em>
+import { events, Event, Job } from "@brigadecore/brigadier"
+
+const localPath = "/workspaces/brigade"
+
+events.on("brigade.sh/github", "push", async event => {
+  let test = new Job("test", "debian:latest", event)
+  test.primaryContainer.sourceMountPath = localPath
+  test.primaryContainer.workingDirectory = localPath
+  test.primaryContainer.command = ["bash"]
+  test.primaryContainer.arguments = ["-c", "make test"]
+  await test.run()
+})
+
+events.process()
               </pre>
             </li>
             <li data-orbit-slide="headline-2">
               <pre>
-<em>// Updating a cosmosDB database</em>
-const { events, Job } = require("brigadier")
-
-events.on("exec", (e, p) => {
-  var mongo = new Job("update-db", "mongo:3.2")
-  mongo.storage.enabled = false
-  mongo.tasks = [
-      dbCmd(p, `db.mydb.insert(${e.payload})`)
-  ]
-  mongo.run()
-})
+<em>// Update a CosmosDB database using JavaScript</em>
+const { events, Job } = require("@brigadecore/brigadier");
+
+events.on("brigade.sh/cli", "exec", async event => {
+  let mongo = new Job("update-db", "mongo:3.2", event);
+  mongo.primaryContainer.environment = {
+    COSMOS_NAME: event.project.secrets.cosmosName,
+    COSMOS_KEY: event.project.secrets.cosmosKey
+  };
+  mongo.primaryContainer.command = ["bash"];
+  mongo.primaryContainer.arguments = [
+    "-c",
+    dbCmd(event.project, `db.mydb.insert(${event.payload})`)
+  ];
+  await mongo.run();
+});
```

Comment:
I worry about the optics of using (or continuing to use) an Azure service as an example. Already most maintainers are MS employees and all _active_ maintainers are MS employees. In the interest of promoting vendor neutrality, which I think is the right thing for a CNCF project to do, I would personally want to see a different example here.

Reply:
Fair enough.  Updated example to just be a generic MongoDB handler.